### ISSUE TITLE
Enable acceptance tests

### ISSUE THREAD

**Issue by bastelfreak** [voxpupuli/puppet-cassandra#431]
this module ships some acceptance tests. we need to review them and enable them on travis.

**Comment by dallinb** [comment]
@bastelfreak I'm probably going to be the first to admit that the acceptance tests were pretty hard to follow and I wrote them.  Also the testing framework seems to have changed radically since the tests were initially written (not to mention the supported Puppet version).  Are there any examples of Vox Pupuli modules that implement acceptance tests and if so I will have a go at writing some for this module.
